cloudformation/OpenMapKitServer.template.js

```diff
@@ -100,15 +132,28 @@ const Resources = {
       Properties: {
         IamInstanceProfile: cf.ref('OpenMapKitServerEC2InstanceProfile'),
         ImageId: 'ami-0565af6e282977273',
-        InstanceType: 'c5d.large',
+        InstanceType: 'c5a.large',
+        BlockDeviceMappings: [
+          {
+            DeviceName: '/dev/sda1',
+            Ebs: {
+              DeleteOnTermination: true,
+              Encrypted: false,
+              Iops: 3000,
+              Throughput: 125,
+              VolumeSize: 100,
```

Comment:
I don't think its necessary for all OMK servers to have a large data directory by default, and doing so will incur unnecessary costs. this should work as a stop-gap measure for the tz server but I'm not sure this is the best solution yet to merge to master.

Reply:
I can parameterize VolumeSize and we can customize the value every run.